axiom/nr_log_events.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nr_axiom.h"
+
+#include <math.h>
+#include <stdlib.h>
+
+#include "nr_log_events.h"
+#include "nr_analytics_events_private.h"
+#include "util_memory.h"
+
+bool nr_log_events_add_event(nr_analytics_events_t* events,
+                             const nr_log_event_t* event,
+                             nr_random_t* rnd) {
+  bool events_sampled = false;
+  char* log_event_json = NULL;
+  nr_analytics_event_t* log_event;
```

Comment:
Set `log_event` to NULL to initialize?

Reply:
Adressed in b339a7ace1c663c9fd0b3539ad6da613e87c7ad3